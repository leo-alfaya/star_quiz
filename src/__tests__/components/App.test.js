import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import App from '../../components/App';

describe('</App>', () => {
    test('App should mount without errors', () => {
        const div = document.createElement('div')
        ReactDOM.render(<App />, div);
    });

    test('App should render without errors', () => {
        const wrapper = shallow(<App />);
        expect(wrapper).toMatchSnapshot();
    });
})
