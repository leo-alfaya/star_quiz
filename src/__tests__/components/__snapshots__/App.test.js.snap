// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`</App> App should render without errors 1`] = `
<div
  className="App"
>
  <WithStyles(CssBaseline) />
  <BrowserRouter>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/game"
    />
  </BrowserRouter>
</div>
`;
